generators/ansible-do/templates/ansible/stack/plays/awscli.yml

```diff
@@ -8,5 +8,7 @@
 
 - name: Install aws-cli
   hosts: awscli
+  collections:
```

Comment:
I don't know where you found this syntax.
I have found instead that the FQN of the role should be used. see https://docs.ansible.com/ansible/latest/collections_guide/collections_using_playbooks.html

Reply:
Ok, changes has been made.

Reply:
Tried using the module syntax but did not worked

Reply:
The syntax you proposed is only usable for module and therefore doesn't work for a role wich we have, it is technically possible to change a role into a module to some degrees but just for syntax preferences it is too much work for what it would benefit.